spaceship.zsh

```diff
@@ -41,6 +41,12 @@ SPACESHIP_GIT_UNPUSHED="${SPACESHIP_GIT_UNPUSHED:-⇡}"
 SPACESHIP_TIME_SHOW="${SPACESHIP_TIME_SHOW:-false}"
 SPACESHIP_TIME_12HR="${SPACESHIP_TIME_12HR:-false}"
 
+# BATTERY
+SPACESHIP_BATTERY_SHOW="${SPACESHIP_BATTERY_SHOW:-false}"
```

Comment:
Also, would be great if percentage would be hidden as default, but theme would show user a reminder to connect charger when the battery is charging out.